### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by AlexSavinskaya** [getredash/redash#6482] [General]
Hi!
I cant find the information about prices for redash. One thing I found out is the price 49$ per month but there is no details about included options. Could anyone help me with it?
Thanks

**Comment by justinclift** [comment]
Sure.  This is an Open Source project that *used* to have a hosted option available.

That hosted version has been shut down, so if you want to run Redash then you'll need to either set it up yourself or get someone else to. :smile:
